docker-compose.prod.yml

```diff
@@ -0,0 +1,60 @@
+version: "3"
+
+services:
+  oba_bundler:
+    build: ./bundler
+    volumes:
+      - ./bundle:/bundle
+    environment:
+      - GTFS_URL
+
+  oba_database:
+    image: mysql:8.3
+    container_name: oba_database
+    environment:
+      MYSQL_ROOT_PASSWORD: Ins3cure!
+      MYSQL_DATABASE: oba_database
+      MYSQL_USER: oba_user
+      MYSQL_PASSWORD: oba_password
+    ports:
+      - "3306:3306"
+    volumes:
+      - type: volume
+        source: mysql-data
+        target: /var/lib/mysql
```

Comment:
what will happen to the mysql data between updates?

Reply:
I don't think this will interfere with MySQL data, which is stored in persistent volume `mysql-data`, It is still the same image with the same volume.

Reply:
mysql data will not be affected, docker will still use `mysql-data` volume, it will be the same image with the same data, see https://docs.docker.com/storage/volumes/#use-a-volume-with-docker-compose